plugins/linux-capture/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ set(linux-capture_LIBRARIES
 
 option(ENABLE_PIPEWIRE "Enable PipeWire support" ON)
 if(ENABLE_PIPEWIRE)
-	find_package(PipeWire 0.3.33 QUIET)
+	find_package(PipeWire 0.3.32 QUIET)
```

Comment:
Could you please squash this commit with 8784a167fc7ae9537aeb2b89b93b9c0d57c169ae ?

Reply:
I would prefer not to squash those. This way we can this spring just do a git revert "Fix ubuntu 21.04 build" and everyone knows, this commit shouldn't have happened, but compatibility sth sth.

Reply:
Alright then